client/src/components/AuthMenu/AuthMenu.tsx

```diff
@@ -41,6 +49,8 @@ const AuthMenu = (): JSX.Element => {
         getContentAnchorEl={null}
       >
         <MenuItem onClick={handleLogout}>Logout</MenuItem>
+        <MenuItem onClick={handleProfile}>Profile</MenuItem>
+        <MenuItem onClick={handlemybookings}>mybookings</MenuItem>
```

Comment:
Would be interesting to see if we can do this with the Link component from Router route dom and MUI

Reply:
These are temporary links, I will remove them.